README.md

```diff
@@ -39,6 +39,9 @@ array of dictionaries for slave nodes
 - name: slave1                   # string to identify the node
   remotefs: /usr/local/jenkins   # home directory of jenkins user on the node
   host: slave1.example.com       # FQDN or IP address
+  labels:
```

Comment:
would you please document the variable using a table? like role variables.

| Key | Value | Mandatory |
|-|-|
| `foo` | string of name of this | yes |
| `bar` | list of name of that | no |